src/providers/object-provider.js

```diff
@@ -193,8 +194,10 @@ export default class YamcsObjectProvider {
         const operation = 'parameters?details=yes&limit=1000';
         const parameterUrl = this.url + 'api/mdb/' + this.instance + '/' + operation;
         const url = this.#getMdbUrl('space-systems');
-        const spaceSystems = await accumulateResults(url, {}, 'spaceSystems', []);
-        const parameters = await accumulateResults(parameterUrl, {}, 'parameters', []);
+        const requestOptions = await this.dictionaryRequestCacheStrategyPromise;
```

Comment:
Should we have some other function like 'buildRequestOptions' that does this? I mean, we can always add it down the road, but it seems weird just assigning directly from the cache strategy promise.

Reply:
i hear you. we could change it now or wait until there are other requests. this one is very specific to loading the dictionary so i think we may be okay.